lms/djangoapps/support/management/commands/get_fbe_info_for_all_courses.py

```diff
@@ -0,0 +1,59 @@
e-v1:LouvainX+Louv12x_E+1T2018", "course-v1:LouvainX+Louv12x_E+2T2018", "course-v1:LouvainX+Louv13x+1T2017", "course-v1:LouvainX+Louv13x+1T2018", "course-v1:LouvainX+Louv13x+1T2019", "course-v1:LouvainX+Louv13x+1T2020", "course-v1:LouvainX+Louv13x+3T2017", "course-v1:LouvainX+Louv13x+3T2018", "course-v1:LouvainX+Louv13x+3T2019", "course-v1:LouvainX+Louv14x+1T2017", "course-v1:LouvainX+Louv14x+1T2018", "course-v1:LouvainX+Louv14x+1T2019", "course-v1:LouvainX+Louv14x+1T2020", "course-v1:LouvainX+Louv14x+2T2017", "course-v1:LouvainX+Louv14x+2T2018", "course-v1:LouvainX+Louv14x+2T2019", "course-v1:LouvainX+Louv15x+1T2018", "course-v1:LouvainX+Louv15x+3T2018", "course-v1:LouvainX+Louv15x+3T2019", "course-v1:LouvainX+Louv16x+1T2017", "course-v1:LouvainX+Louv16x+1T2019", "course-v1:LouvainX+Louv16x+1T2020", "course-v1:LouvainX+Louv16x+2T2019", "course-v1:LouvainX+Louv16x+3T2017", "course-v1:LouvainX+Louv16x+3T2018", "course-v1:LouvainX+Louv17.1x+3T2017", "course-v1:LouvainX+Louv17.1x+3T2018", "course-v1:LouvainX+Louv17.1x+3T2019", "course-v1:LouvainX+Louv17.2x+1T2018", "course-v1:LouvainX+Louv17.2x+1T2019", "course-v1:LouvainX+Louv17.2x+1T2020", "course-v1:LouvainX+Louv17x+3T2016", "course-v1:LouvainX+Louv18x+2T2017", "course-v1:LouvainX+Louv18x+2T2019", "course-v1:LouvainX+Louv18x+3T2016", "course-v1:LouvainX+Louv18x+3T2018", "course-v1:LouvainX+Louv19x+2T2018", "course-v1:LouvainX+Louv19x+2T2019", "course-v1:LouvainX+Louv19x+3T2016", "course-v1:LouvainX+Louv20x+1T2018", "course-v1:LouvainX+Louv20x+1T2019", "course-v1:LouvainX+Louv20x+1T2020", "course-v1:LouvainX+Louv20x+3T2017", "course-v1:LouvainX+Louv20x+3T2018", "course-v1:LouvainX+Louv20x+3T2019", "course-v1:LouvainX+Louv20x+3T2020", "course-v1:LouvainX+Louv21x+1T2020", "course-v1:LouvainX+Louv22x+1T2019", "course-v1:LouvainX+Louv22x+1T2020", "course-v1:LouvainX+Louv22x+3T2019", "course-v1:LouvainX+Louv23x+1T2018", "course-v1:LouvainX+Louv23x+1T2019", "course-v1:LouvainX+Louv23x+3T2019", "course-v1:LouvainX+Louv24x+1T2020", "course-v1:LouvainX+Louv24x+3T2018", "course-v1:LouvainX+Louv24x+3T2019", "course-v1:LouvainX+Louv25x+3T2020", "course-v1:LouvainX+Louv26x+1T2020", "course-v1:LouvainX+Louv27x+1T2020", "course-v1:LouvainX+Louv27x+3T2019", "course-v1:LouvainX+Louv28x+3T2020", "course-v1:LouvainX+Louv2x+1T2016", "course-v1:LouvainX+Louv2x+1T2017", "course-v1:LouvainX+Louv2x+1T2018", "course-v1:LouvainX+Louv2x+1T2019", "course-v1:LouvainX+Louv2x+1T2020", "course-v1:LouvainX+Louv2x+2T2016", "course-v1:LouvainX+Louv3x+1T2016", "course-v1:LouvainX+Louv3x+1T2017", "course-v1:LouvainX+Louv3x+1T2018", "course-v1:LouvainX+Louv3x+1T2019", "course-v1:LouvainX+Louv3x+1T2020", "course-v1:LouvainX+Louv3x+2T2016", "course-v1:LouvainX+Louv3x+2T2017", "course-v1:LouvainX+Louv3x+2T2018", "course-v1:LouvainX+Louv3x+2T2019", "course-v1:LouvainX+Louv3x+3T2015", "course-v1:LouvainX+Louv3x+3T2019", "course-v1:LouvainX+Louv3x+3T2020", "course-v1:LouvainX+Louv3x_E+1T2018", "course-v1:LouvainX+Louv3x_E+2T2020", "course-v
```

Comment:
would we be able to get the csv from where it gets written to?

Reply:
It should just be on the server, so yes? I'll merge, and then fast-follow if the devOps team says no.